notebooks/book1/02/gamma_dist_plot.ipynb

```diff
@@ -0,0 +1,611 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #13.            plt.ylabel("$pdf$")
</pre><p>Use $p(x)$ instead of pdf.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>